src/main/java/seedu/address/logic/commands/SplitCommand.java

```diff
@@ -0,0 +1,107 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MONEY_OWED;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.MoneyOwed;
+import seedu.address.model.person.Person;
+
+/**
+ * Splits the sum of money owed among a group of person using the displayed
+ * index from the address book.
+ */
+public class SplitCommand extends Command {
+    public static final String COMMAND_WORD = "split";
+    public static final Float MINIMUM_SPLIT_AMOUNT = (float) 0.01;
+    public static final String MESSAGE_INVALID_AMOUNT =
+            "Amount after splitting should be more than $0.01!";
+    public static final String MESSAGE_MISSING_AMOUNT =
+            "Please enter an amount that you want to split!";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Splits the sum of money owed among a group of person "
+            + "using the displayed index from the address book.\n"
+            + "Parameters: at least one INDEX (must be a positive integer) "
+            + PREFIX_MONEY_OWED + "MONEY_OWED "
+            + "Example: " + COMMAND_WORD + " 1 2 "
+            + PREFIX_MONEY_OWED + "4.50";
+    private final List<Index> indexListToSplit;
+    private final MoneyOwed totalOwed;
+    /**
+     * Returns a new SplitCommand object that takes in a list of index
+     * and a MoneyOwed object.
+     * @param indexListToSplit
+     * @param totalOwed
+     */
+    public SplitCommand(List<Index> indexListToSplit, MoneyOwed totalOwed) {
+        this.indexListToSplit = indexListToSplit;
+        this.totalOwed = totalOwed;
+    }
+    /**
+     * Splits the total amount of a group of people.
+     * @param totalAmount
+     * @param numPeople
+     * @return the split amount
+     */
+    public static Float getSplitAmount(Float totalAmount, int numPeople) {
```

Comment:
Suppose there is some amount whose rounded split values do not sum up to the original value, i.e. $1/3 ~= $0.33, but $0.33*3 = $0.99, this can cause there to be some deviation in actual values in the long run. Doesn't seem like this case is handled?

Reply:
In this case, I decided to round it off to 2 dp, because for example the debt is 0.33333 and after paying off 0.33, you are left with 0.00333, our UI will show he owes you 0.00(2dp), when 0.00 should be you do not owe each other anything.